.github/pull_request_template.md

```diff
@@ -1,31 +1,79 @@
-# Description
+# Pull Request OpenBB
 
-- [ ] Summary of the change / bug fix.
+## The OpenBBTerminal
+
+<details>
+<summary> Pull Request the OpenBBTerminal</summary>
+
+### Description
+
+- [ ] Summary of the change/ bug fix.
 - [ ] Link # issue, if applicable.
 - [ ] Screenshot of the feature or the bug before/after fix, if applicable.
 - [ ] Relevant motivation and context.
 - [ ] List any dependencies that are required for this change.
 
+### How has this been tested?
 
-# How has this been tested?
+- Please describe the tests that you ran to verify your changes.
+- Provide instructions so we can reproduce.
+- Please also list any relevant details for your test configuration.
 
-* Please describe the tests that you ran to verify your changes.
-* Provide instructions so we can reproduce.
-* Please also list any relevant details for your test configuration.
 - [ ] Make sure affected commands still run in terminal
 - [ ] Ensure the SDK still works
 - [ ] Check any related reports
 
+### Checklist
 
-# Checklist:
 
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
 - [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
 - [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
```

Comment:
Err... shouldn't we provide the link to our Platform guidelines?

Reply:
I think we will still need some different guidelines for the Terminal since it will have its own particularities (despite using the Platform).